files/ko/web/javascript/reference/global_objects/string/raw/index.html

```diff
@@ -0,0 +1,121 @@
+---
+title: String.raw()
+slug: Web/JavaScript/Reference/Global_Objects/String/raw
+tags:
+- ECMAScript 2015
+- JavaScript
+- Method
+- Reference
+- String
+- Polyfill
+translation_of: Web/JavaScript/Reference/Global_Objects/String/raw
+---
+<div>{{JSRef}}</div>
+
+<p>
+    <strong><code>String.raw()</code></strong> 메서드는 <a
+        href="/ko/docs/Web/JavaScript/Reference/Template_literals">템플릿리터럴</a>의 태그 함수입니다. 이는 Pyhon의 <code>r</code>접두사 또는
+    C#의 문자열 리터럴의
+    <code>@</code> 접두사와 유사합니다. (그러나 동일 하지는 않습니다. 이 문제에 관해서는 여기 <a
+        href="https://bugs.chromium.org/p/v8/issues/detail?id=5016">이슈</a>를 참조하십시오.) 이 메서드는 템플릿 리터럴의 윈시 문자열을 가져오는데
+    사용됩니다.
+    즉, 대체(예: <code>${foo}</code>)는 처리되지만 이스케이프(예: <code>\n</code>)는 처리되지 않습니다.
+</p>
+<p></p>
+
+<p> </p>
+
+<div>{{EmbedInteractiveExample("pages/js/string-raw.html")}}</div>
+
+
+
+<p> </p>
+
+<h2 id="구문">구문</h2>
+
+<pre class="brush: js">String.raw(callSite, ...substitutions)
+
+String.raw`templateString`
+</pre>
+
+<h3 id="매개변수">매개변수</h3>
+
+<dl>
+    <dt><code><var>callSite</var></code></dt>
+    <dd>정해진 형식의 템플릿 호출 개체 예:
+        <code>{ raw: ['foo', 'bar', 'baz'] }</code>.
+    </dd>
+    <dt><code><var>...substitutions</var></code></dt>
+    <dd><code><var>callSite</var></code> 사이사이에 대체값으로 포함됩니다. 예: <code>(['foo','baz'],'bar') => 'foobarbaz'</code></dd>
```

Comment:
```suggestion
    <dd>대체값을 포함합니다.</dd>
```

원문에서는 `Contains substitution values.` 이렇게 돼있습니다. 가급적 원문과 동일한 내용으로 부탁드려요 !